scripts/throughput_benchmarks.py

```diff
@@ -280,12 +286,17 @@ def run_benchmark(
         "avg_per_session_sampling_throughput": num_sampled_tokens
         / (elapsed * avg_completion_time / (avg_prefill_time + avg_completion_time))
         / concurrency,
+        "avg_request_throughput": n / elapsed,
         "avg_inter_token_latency": sum(inter_token_latency) / n,
```

Comment:
what about percentiles for this? also i wonder if we can have percentiled numbers for throughput numbers?

Reply:
+1 I think throughput is more easily interpretable to users?

Reply:
I've added percentiles for inter-token latency + time to first token, not sure how to get percentiles for throughput (since I think it's a single value for the entire test)